### PR TITLE
Update disguise shortDesc

### PR DESCRIPTION
**Pull request by Enrii17**

### CODE REVIEW

**Comment by Zarel** [comment]
We still need the old desc for Gen 7. Can you add it in there?

File: data/abilities.js [hunk 1]
```diff
@@ -744,7 +744,7 @@ let BattleAbilities = {
 	},
 	"disguise": {
 		desc: "If this Pokemon is a Mimikyu, the first hit it takes in battle deals 0 neutral damage. Its disguise is then broken, it changes to Busted Form, and it loses 1/10 of its max HP. Confusion damage also breaks the disguise.",
-		shortDesc: "If this Pokemon is a Mimikyu, the first hit it takes in battle deals 0 neutral damage.",
+		shortDesc: "If this Pokemon is a Mimikyu, first hit 0 damage. Breaks disguise losing 10% HP",
```
Comment (Zarel) [inline data/abilities.js]:
```suggestion
		shortDesc: "(Mimikyu only) The first hit it takes is blocked, and it takes 10% HP damage instead.",
```

Comment (Enrii17) [reply data/abilities.js]: Sure, do we need it for every ability that changed in Gen 8 or just Disguise?

Comment (Zarel) [reply data/abilities.js]: We've already done it for other Abilities:

https://dex.pokemonshowdown.com/abilities/moody

Comment (Zarel) [reply data/abilities.js]: Feel free to do it for any ability we've missed, though?

**Comment by Zarel** [comment]
You still haven't fixed it in past gens...

**Comment by Enrii17** [comment]
> You still haven't fixed it in Gen 7...

oh sorry, my internet was extremely bad past few days and I forgot about past gens, will do it now

File: data/mods/gen7/abilities.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ let BattleAbilities = {
 	"disguise": {
 		inherit: true,
 		desc: "If this Pokemon is a Mimikyu, the first hit it takes in battle deals 0 neutral damage. Its disguise is then broken and it changes to Busted Form. Confusion damage also breaks the disguise.",
+		shortDesc: "(Mimikyu only) First hit deals 0 damage, breaks disguise.",
```
Comment (Zarel) [inline data/mods/gen7/abilities.js]:
```suggestion
		shortDesc: "(Mimikyu only) The first hit it takes is blocked and deals 0 damage instead.",
```

File: data/abilities.js [hunk 3]
```diff
@@ -744,7 +744,7 @@ let BattleAbilities = {
 	},
 	"disguise": {
 		desc: "If this Pokemon is a Mimikyu, the first hit it takes in battle deals 0 neutral damage. Its disguise is then broken, it changes to Busted Form, and it loses 1/10 of its max HP. Confusion damage also breaks the disguise.",
```
Comment (Zarel) [inline data/abilities.js]:
```suggestion
		desc: "If this Pokemon is a Mimikyu, the first hit it takes in battle deals 0 neutral damage. Its disguise is then broken, it changes to Busted Form, and it loses 1/8 of its max HP. Confusion damage also breaks the disguise.",
```

File: data/abilities.js [hunk 4]
```diff
@@ -744,7 +744,7 @@ let BattleAbilities = {
...
-		shortDesc: "If this Pokemon is a Mimikyu, the first hit it takes in battle deals 0 neutral damage.",
+		shortDesc: "(Mimikyu only) The first hit it takes is blocked, and it takes 10% HP damage instead.",
```
Comment (Zarel) [inline data/abilities.js]:
```suggestion
		shortDesc: "(Mimikyu only) The first hit it takes is blocked, and it takes 1/8 HP damage instead.",
```

**Pull request closed without merging**